prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -269,15 +271,17 @@ public async Task SendEnrolleeRenewalEmails()
                     await Send(email);
                 }
             }
-        }
 
+            return enrollees.Select(e => e.Id);
```

Comment:
In my testing, I got 20 event entries but no emails. I believe this should return the enrollee IDs that has email sent, not every enrollee from the query. It seems SendEnrolleeUnsignedToaReminderEmails has the similar issue as well.

Reply:
Good catch